src/VisualStudio/Core/Test/Utilities/VsNavInfoHelpers.vb

```diff
@@ -0,0 +1,64 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Runtime.CompilerServices
+Imports Microsoft.VisualStudio.Shell.Interop
+
+Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Utilities.VsNavInfo
+    Friend Module VsNavInfoHelpers
+
+        Public Sub IsOK(comAction As Func(Of Integer))
```

Comment:
This is also existing code that was moved to promote sharing and demote copying and pasting. I'll take a look at other usages.